astropy/table/table.py

```diff
@@ -3077,6 +3077,54 @@ def _replace_cols(self, columns):
 
         self.columns = columns
 
+    def update(self, other, column_conflicts='warn',
```

Comment:
Since this is meant to be analogous to `dict.update`, I think we can make the default be `column_conflicts='silent'`.

Reply:
Should `metadata_conflicts='silent'` be the default too? That might be more consistent with `dict.update()` but less consistent with some of the table stacking operations.

Reply:
I think so.